tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Basic tests scenarios for snapshot save/restore."""
+
+import logging
+import platform
+import pytest
+from conftest import _test_images_s3_bucket
+from framework.artifacts import ArtifactCollection, ArtifactSet
+from framework.matrix import TestMatrix, TestContext
+from framework.builder import MicrovmBuilder, SnapshotBuilder, SnapshotType
+import host_tools.network as net_tools  # pylint: disable=import-error
+
+
+def _test_5_seq_snapshots(context):
```

Comment:
I would name this `_test_multiple_seq_snapshots` and take the number as an argument.